File: 1247.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        while (sc.hasNextInt()) {
            int d = sc.nextInt();
            int vf = sc.nextInt();
            int vg = sc.nextInt();

            double tempoF = 12.0 / vf;
            double tempoG = Math.sqrt(d * d + 144.0) / vg;

            if (tempoG <= tempoF)
                System.out.println("S");
            else
                System.out.println("N");
        }

        sc.close();
    }
}

File: 1247.c
#include <math.h>
#include <stdio.h>

int main() {
    int d, vf, vg;
    float tempoF, tempoG;

    while(scanf("%d %d %d", &d, &vf, &vg) == 3) {
        tempoF = 12.0 / vf;
        tempoG = sqrt((d * d) + 144.0) / vg;

        if(tempoG <= tempoF)
            printf("S\n");
        else
            printf("N\n");
    }

    return 0;
}
